test/services/solana_rpc_client_test.rb

```diff
@@ -24,4 +24,25 @@ class SolanaRpcClientTest < ActiveSupport::TestCase
       assert_equal 251, response.result["epoch"]
     end
   end
+
+  test "#network_client returns correct client" do
+    # No token, for privacy reason.
+    vcr_cassette(@namespace, __method__) do
+      rpc_client = SolanaRpcClient.new
+      mainnet = rpc_client.network_client("mainnet")
+
+      # returns same client when same network used again
+      assert_equal rpc_client.network_client("mainnet"), mainnet
+      # returns different client when different network used
+      refute rpc_client.network_client("pythnet") == mainnet
```

Comment:
When we did `rpc_client.network_client("pythnet")` here, then the client will be `nil`. Should it be like this?

Reply:
rpc_client.network_client("pythnet") returns `SolanaRpcRuby::MethodsWrapper` object, which doesnt have `client` method/attribute
If you run
```
rpc = SolanaRpcClient.new
rpc.network_client("pythnet")
rpc.client
```
It will return correct wrapper